_pydevd_sys_monitoring/_pydevd_sys_monitoring.py

```diff
@@ -205,6 +219,17 @@ def _get_unhandled_exception_frame(depth: int) -> Optional[FrameType]:
                 if f_back.f_code.co_name.startswith(("run", "_run")):
                     break
 
+            elif name == "<frozen runpy>":
+                if f_back.f_code.co_name.startswith(("run", "_run")):
+                    break
+
+            elif name == "runpy":
+                if f_back.f_code.co_name.startswith(("run", "_run")):
+                    break
+            
+            elif py_db.get_file_type(f_back) is PYDEV_FILE:
```

Comment:
I noticed a pattern and thought maybe this would catch other things that were missed in the if statements above. But I don't recall this being necessary. I'll remove it and see if everything still passes.